helm/kfp-operator/values.yaml

```diff
@@ -51,13 +51,22 @@ manager:
   webhookServicePort: 9443
   runcompletionWebhook:
     servicePort: 8082
-    endpoints: []
+    endpoints:
+      - host: kfp-operator-run-completion-event-trigger.kfp-operator-system
+        path: ""
+        port: 50051
 
 statusFeedback:
   enabled: false
-  webhook:
-    endpoint: '/'
-    port: 12000
+  runcompletionEventTrigger:
```

Comment:
I see webhook was wrong before but this shouldn't be inside statusFeedback, should be part of a triggers section or something and default again should be it isn't deployed.